openshift-hack/e2e/annotate/generated/zz_generated.annotations.go

```diff
@@ -213,6 +213,8 @@ var Annotations = map[string]string{
 
 	"[sig-api-machinery] OpenAPIV3 should round trip OpenAPI V3 for all built-in group versions": " [Suite:openshift/conformance/parallel] [Suite:k8s]",
 
+	"[sig-api-machinery] OrderedNamespaceDeletion namespace deletion should delete pod first [Feature:OrderedNamespaceDeletion] [FeatureGate:OrderedNamespaceDeletion] [Beta]": " [Suite:openshift/conformance/parallel] [Suite:k8s]",
+
```

Comment:
This new test is failing because it's testing a new feature that is disabled by default. We'll need to disable it. Here's the commit for 4.18 that you could use as an example:

https://github.com/openshift/kubernetes/pull/2248/commits/579a60270ce6759d8389efeee2f948c44751e115